golem-service-base/src/model.rs

```diff
@@ -3084,6 +3098,17 @@ impl RoutingTable {
             self.number_of_shards.value,
         ))
     }
+
+    pub fn random(&self) -> Option<&Pod> {
+        use rand::seq::IteratorRandom;
```

Comment:
Should this be within this function?

Reply:
i can move it to common imports, but is it problem when it is here?